### PR TITLE
Base styles

### PR DESCRIPTION
**Pull request by jmripper**

### CODE REVIEW

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@jmripper Can you write a description and reference the issue this is for?

**Comment by jmripper** [comment]
Created CSS styles for the colors and typography from the style guide in Figma.

OG Issue: https://github.com/ZakkMan/modern-trousseau/issues/44

@ZakkMan

File: client/src/App.css [hunk 1]
```diff
@@ -1,38 +1,273 @@
...
+  font-family: "Raleway", sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 24px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
+}
+
+.body1 {
```
Comment (0xZakk) [inline client/src/App.css]: Can you come up with more semantic names? We typically never want to put numbers in class names

File: client/src/components/NavMenu/NavMenu.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+.nav-container {
+        background-color: #FAE5DC;
+        display: flex;
+        flex-flow: column;
+        justify-content: flex-start;
+        align-content: space-around;
+        align-items: center;
```
Comment (0xZakk) [inline client/src/components/NavMenu/NavMenu.css]: Looks like your nav menu stuff is in this PR too. Can you remove those changes from here too?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@jmripper This looks great! Really great work - just a few changes below

File: client/src/components/NavMenu/NavMenu.css (hunk 2 above)
Comment (jmripper) [reply client/src/components/NavMenu/NavMenu.css]: @ZakkMan I removed the files from the PR.

**Review by 0xZakk — APPROVED** [review]
@jmripper looks good!

**Pull request merged**